unlock-js/src/__tests__/v13/initializeTemplate.test.js

```diff
@@ -0,0 +1,66 @@
+import WalletService from '../../walletService'
+import v13 from '../../v13'
+import { ZERO } from '../../constants'
+
+let walletService
+
+const transaction = {
```

Comment:
I took an extra effort to NOT use nock in this as I really want to get rid of it inside of our unlock-js unit tests. 
I believe it is cleaner this way